packages/api/app/Models/User.js

```diff
@@ -88,7 +93,7 @@ class User extends Model {
 	}
 
 	getFullName({ first_name, last_name }) {
-		return `${first_name} ${last_name}`;
+		return last_name !== '' ? `${first_name} ${last_name}` : first_name;
```

Comment:
Será que assim também conseguimos resolver?
```suggestion
		return `${first_name} ${last_name}`.trim();
```